utils/update_translation_stats.py

```diff
@@ -88,47 +86,46 @@ def generate_translation_stats(po_dir, output_file):
         column_counter = 0
         total_column = 0
         translated_column = 0
-        while column_counter < len(header_entries):
+        for column_counter in range(len(header_entries)):
             if header_entries[column_counter].strip() == 'Total Source Words':
                 total_column = column_counter
             elif header_entries[column_counter].strip() == 'Translated Source Words':
                 translated_column = column_counter
-            column_counter = column_counter + 1
 
         # Now do the actual counting for the current textdomain
         for line in result:
             cells = line.split(',')
             po_filename = cells[0]
             if po_filename.endswith('.po'):
-                entry = TranslationStats()
                 locale = regex_po.match(po_filename).group(1)
-                if locale in locale_stats:
-                    entry = locale_stats[locale]
-                entry.total = entry.total + int(cells[total_column])
-                entry.translated = entry.translated + \
-                    int(cells[translated_column])
+                entry = locale_stats[locale]
+                entry.total += int(cells[total_column])
+                entry.translated += int(cells[translated_column])
                 if entry.translated > entry.total:
-                    print('Error! Translated ' + str(entry.translated) + ' (+' + (cells[
-                          translated_column]) + ') is bigger than the total of ' + str(entry.total) + '(' + (cells[total_column]) + ')\n' + line)
+                    print('Error! Translated ' + str(entry.translated) + ' (+' +
+                          cells[translated_column] + ') is bigger than the total of ' +
+                          str(entry.total) + '(' + cells[total_column] + ')\n' + line)
                     sys.exit(1)
                 locale_stats[locale] = entry
 
     print('\n\nLocale\tTotal\tTranslated')
     print('------\t-----\t----------')
 
     # The total goes in a [global] section and is identical for all locales
-    result = '[global]\n'
-    result = '{}total={}\n\n'.format(
-        result, list(locale_stats.values())[0].total)
+    result = '''# This file is managed by utils/update_translations_stats.py
+
+[global]
+'''
```

Comment:
(multiline `'''` strings just make python's ugly indenting even worse… but of course that's just my taste)